### ISSUE TITLE
Rundeck Pager Duty Plugin

### ISSUE THREAD

**Issue by utsawtiwari** [rundeck/docs#1574]
I'm running Rundeck as a Kubernetes pod with a MySQL database and have installed a PagerDuty plugin. However, when I restart the deployment, the plugin is removed. Should I consider running Rundeck as a StatefulSet to retain plugin data? Does the plugin information not persist in the database?

**Comment by fdevans** [comment]
Yes. you would need to find a way to add or persist your plugins. 

Always best to ask technical questions like this on the main repository or our community forums.  The Docs repo issues are just for documentation needs.
